source/includes/steps-install-mongodb-on-amazon-tarball.yaml

```diff
@@ -34,27 +34,27 @@ level: 4
 ref: ensure-binaries-in-path
 content: |
 
-   The MongoDB binaries are in the ``<mongodb-install-directory>/bin/``
+   The MongoDB binaries are in the ``<mongodb-install-directory>/bin``
    directory. To avoid having to specify the path to the MongoDB
-   binaries, you can modify your user's ``PATH`` environment variable
-   to include this directory. For example, you can add the following
-   line to your shell's initialization script (e.g. {{bashrc}}):
+   binaries, add the contents of the
+   ``<mongodb-install-directory>/bin/`` directory to a directory in the
```

Comment:
bleh. I don't like this, and if we're generally certain that the names of tarballs don't change, we could be a lot more specific.

Unfortunately, AL1 and AL2 have different tarball names so it would require tabbing to make this easier. But the entire thing could be 100% copy-pastable w/o any additional user edits with a few small tweaks.